script/Arcueid/Day 7/07_00_ARC07_3_1.txt

```diff
@@ -110,7 +110,7 @@ Geez, what kind of person do you take me for?! Are you misunderstanding somethin
 [sha:cf209a990933d095c5dc3326c5e9107a9aa36d8b]{
 -- Page 507, Offset 17831.
 -- 「大丈夫、そこは完璧、ブレてない。アルクェイドの事は、手加減のできないやりすぎ吸血鬼だと思ってる」
-"Don't worry, I know exactly what to take you for. A vampire who goes too far and doesn't know how to hold back."
+"Don't worry, I know exactly who. You're a vampire who goes too far and doesn't know how to hold back."
```

Comment:
Thought I reverted this alr

Reply:
What's the story behind this one? Is this supposed to be part of this PR? If you're trying to undo previous pr edit, I think the original was _who you are_. Simply who doesn't really flow well from the prev line.

Reply:
Not sure why this was marked resolved, its still wrong.

Reply:
I just corrected it in my text editor of what I saw on the main branch script. Unless I somehow fucked it somehow and just saw my previous edit.

Reply:
Main currently has this line as:
> "Don't worry, I know exactly what to take you for. A vampire who goes too far and doesn't know how to hold back."

Reply:
Should I go with that version of the line or the original that Ross referenced with "...who you are." to which would it look something like.

"Don't worry, I know exactly who you are. You're a vampire who goes too far and doesn't know how to hold back."?

Reply:
I prefer "what to take you for" personally; I think we discussed this and although it's not necessarily a phrase, I think its a cute response.